roles/node_exporter/tasks/main.yml

```diff
@@ -0,0 +1,32 @@
+- name: 'set node exporter repository'
+  yum_repository:
+    name: prometheus-rpm_release
+    baseurl: https://packagecloud.io/prometheus-rpm/release/el/$releasever/$basearch
+    description: prometheus-rpm_release
+    enabled: True
+    gpgcheck: False
```

Comment:
Oh?

Reply:
but repo_gpgcheck: True (which is an oddity specific to packagecloud.io)